eng/pipelines/templates/steps/build.yml

```diff
@@ -48,12 +48,29 @@ steps:
   - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
     parameters:
       Artifacts: ${{ parameters.Artifacts }}
+  - ${{if and(in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
+    - task: AzureCLI@2
+      inputs:
+        azureSubscription: azure-sdk-tests
+        scriptType: 'bash'
+        scriptLocation: 'inlineScript'
+        inlineScript: |
+          #az account set --name "Azure SDK Engineering System"
+          az account show
+          expiry=`date -u -d "45 minutes" '+%Y-%m-%dT%H:%MZ'`
+          sasToken=$(az storage container generate-sas --account-name azuresdkartifactstest --name azure-sdk-for-js-rush-cache --permissions dlrw --auth-mode login --as-user --expiry $expiry --https-only -o tsv)
```

Comment:
Can we use our existing azuresdkartifacts storage account with a new container? If so you can use the "Azure SDK Artifacts" as the service connection.

Reply:
> Can we use our existing azuresdkartifacts storage account with a new container? If so you can use the "Azure SDK Artifacts" as the service connection.

That's exactly what I want.  I didn't know about `"Azure SDK Artifacts"` service connection.

Reply:
Yes this worked https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4082537&view=logs&j=69104f8e-b88e-5d11-595c-90fd5e8e65ca&t=0504562c-8e1f-55f0-19af-f6f4322d0795&l=679

I will remove "Manual" so we only write to cache in IndividualCI or BatchedCI

Reply:
We already have an azure-sdk-for-js container can we put the cache under that as well?